src/Service.Tests/Unittests/MultiSourceQueryExecutionUnitTests.cs

```diff
@@ -146,15 +121,110 @@ public async Task TestMultiSourceQuery()
             QueryResult queryResult = (QueryResult)result;
             Assert.IsNotNull(queryResult.Data, "Data should be returned for multisource query.");
             IReadOnlyDictionary<string, object> data = queryResult.Data;
-            Assert.IsTrue(data.TryGetValue(queryName1, out object queryNode1), $"Query node for {queryName1} should have data populated.");
-            Assert.IsTrue(data.TryGetValue(queryName2, out object queryNode2), $"Query node for {queryName2} should have data populated.");
+            Assert.IsTrue(data.TryGetValue(QUERY_NAME_1, out object queryNode1), $"Query node for {QUERY_NAME_1} should have data populated.");
+            Assert.IsTrue(data.TryGetValue(QUERY_NAME_2, out object queryNode2), $"Query node for {QUERY_NAME_2} should have data populated.");
 
             ResultMap queryMap1 = (ResultMap)queryNode1;
             ResultMap queryMap2 = (ResultMap)queryNode2;
 
             // validate that the data returend for the queries we did matches the moq data we set up for the respective query engines.
-            Assert.AreEqual("db1", queryMap1[0].Value, $"Data returned for {queryName1} is incorrect for multi-source query");
-            Assert.AreEqual("db2", queryMap2[0].Value, $"Data returned for {queryName2} is incorrect for multi-source query");
+            Assert.AreEqual("db1", queryMap1[0].Value, $"Data returned for {QUERY_NAME_1} is incorrect for multi-source query");
+            Assert.AreEqual("db2", queryMap2[0].Value, $"Data returned for {QUERY_NAME_2} is incorrect for multi-source query");
+        }
+
+        /// <summary>
+        /// Validates successful execution of a query against multiple sources for graphql scenario.
+        /// 1. Tries to use a built schema to execute a query against multiple sources.
```

Comment:
built schema? this sounds like graphql again

Reply:
Missed this sorry when taking the summary over from previous function. fixed.